test/suites/metrics.sh

```diff
@@ -125,6 +125,11 @@ test_metrics() {
   curl -k -s -X GET "https://${metrics_addr}/1.0/metrics" | grep "lxd_filesystem_avail_bytes" | grep "type=\"container\""
   curl -k -s -X GET "https://${metrics_addr}/1.0/metrics?project=default" | grep "lxd_filesystem_avail_bytes" | grep "type=\"container\""
 
+  # API requests metrics should be included
```

Comment:
Can you expand this to include a test for counters going up correctly and for recording bad requests correctly?